source/iodhbwm-templates.sty

```diff
@@ -84,9 +84,20 @@
 \def\iodhbwm@bachelor@type@bos{Bachelor of Science}
 \def\iodhbwm@bachelor@type@boa{Bachelor of Arts}
 
-\def\iodhbwm@thesis@type@ba{Bachelorarbeit}
-\def\iodhbwm@thesis@type@pa{Projektarbeit}
-\def\iodhbwm@thesis@type@sa{Studienarbeit}
+\def\iodhbwm@thesis@type@ba{UNDEFINED LANGUAGE}
+\def\iodhbwm@thesis@type@pa{UNDEFINED LANGUAGE}
+\def\iodhbwm@thesis@type@sa{UNDEFINED LANGUAGE}
+
+\iflanguage{ngerman}{
+	\def\iodhbwm@thesis@type@ba{Bachelorarbeit}
+	\def\iodhbwm@thesis@type@pa{Projektarbeit}
+	\def\iodhbwm@thesis@type@sa{Studienarbeit}
+}{}
+\iflanguage{english}{
+	\def\iodhbwm@thesis@type@ba{bachelor's thesis}
```

Comment:
I've not tested the code yet, but shouldn't these world also start with a capital letter?

Reply:
These variables aren't used very consistent and in the view cases they are used I think it's correct this way